docs/data-sources/resource.md

```diff
@@ -38,16 +38,16 @@ data "twingate_resource" "foo" {
 Read-Only:
 
 - `allow_icmp` (Boolean) Whether to allow ICMP (ping) traffic
```

Comment:
How come no readonly here?

Reply:
in datasource mostly all fields are computed, I think that's why it listed in ReadOnly section

Reply:
oh did you mean, there is no readonly written on this field?

we don't control this, thats doc generation library adds this extra info, maybe for simple fields it's omitted 
as I see the same for: name, address, policy